packages/cli/src/PublicApi/v1/openapi.yml

```diff
@@ -0,0 +1,307 @@
+---
+openapi: 3.0.0
+info:
+  title: Public n8n API
+  description: n8n Public API
```

Comment:
I'd like to make this more substantial for when we have published docs, as this will be the main introduction at the top of the API reference/playground. At minimum, it would be good to outline the API's purpose, scope, and where folks can get help.